src/main/java/com/alesya/mafa/pet/OperationInterface.java

```diff
@@ -97,23 +109,30 @@ public void setScanner(Scanner scanner) {
     }
 
     private Cat createCat() {
-        Cat cat = new Cat(addPet());
+        Cat cat = new Cat(addPet(CAT_BREED));
         System.out.println(HAIR);
         cat.setHair(scanner.nextLine());
         return cat;
     }
 
     private Horse createHorse() {
-        Horse horse = new Horse(addPet());
+        Horse horse = new Horse(addPet(BREED));
```

Comment:
why do you pass static field in parameters?

Reply:
updated